src/server/dflycmd.cc

```diff
@@ -250,7 +250,7 @@ void DflyCmd::Sync(CmdArgList args, ConnectionContext* cntx) {
     TransactionGuard tg{cntx->transaction};
     AggregateStatus status;
 
-    auto cb = [this, &status, replica_ptr](unsigned index, auto*) {
+    auto cb = [this, &status, replica_ptr = replica_ptr](unsigned index, auto*) {
```

Comment:
why do you need `replica_ptr = replica_ptr` ?

Reply:
Because capturing structured bindings is only allowed from c++20